mobsf/StaticAnalyzer/views/android/rules/android_rules.yaml

```diff
@@ -1,3 +1,46 @@
+- id: android_safe_browsing_api_disabled
+  message: >-
+    The app does not use the default SafeBrowsing API. 
+    Handling page navigation should be analized carefully, 
+    as it can load untrusted content.'
+  type: Regex
+  pattern: setSafeBrowsingEnabled\(.*false
+  severity: warning
+  input_case: exact
+  metadata:
+    cvss: ''
+    cwe: ''
+    owasp-mobile: ''
+    masvs: platform-2
+    ref: https://mobile-security.gitbook.io/mobile-security-testing-guide/android-testing-guide/0x05h-testing-platform-interaction#testing-for-url-loading-in-webviews-mstg-platform-2
```

Comment:
Since the gitbook is not versioned and the URLs can change, please use the forked version here

https://github.com/MobSF/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#safebrowsing-api